PagesGallery/PagesGallery.Droid/PagesGallery.Droid.csproj

```diff
@@ -205,7 +212,6 @@
     <AndroidResource Include="Resources\values\styles.xml" />
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-  <Import Project="..\..\.nuspec\Xamarin.Forms.targets" />
```

Comment:
if this project was referencing the XF nuget, it would import the `.targets` (even if it does not do anything for this project at this moment).

to stay future-proof, and make sure that, when we add a special rule in `.targets` it is applied in our tests just like it would in user's projects, please keep this import in.

this comment applies to every platform head where the Import was removed, obviously

Reply:
Fixed.